test/unit/base.test.js

```diff
@@ -37,6 +38,25 @@ describe('CONVERT FUNCTION TESTS ', function() {
         expect(conversionResult.output[0].data).to.have.property('info');
         expect(conversionResult.output[0].data).to.have.property('item');
 
+        done();
+      });
+    });
+    it('[Github #90] - Should create a request using local server instead of global server ' +
+    specPath2, function(done) {
+      Converter.convert({ type: 'file', data: specPath2 }, { schemaFaker: true }, (err, conversionResult) => {
+        let protocol = conversionResult.output[0].data.item[1].request.url.protocol,
```

Comment:
`conversionResult.output[0].data.item[1]` is repeated too often - should be moved to a variable for reuse